MeNext/Playing/Poller.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace MeNext
+{
+	public class Poller
+	{
+		private const int MS_PER_POLL = 200;
+
+		/// <summary>
+		/// Polls the server until we stop polling
+		/// 
+		/// https://developer.xamarin.com/guides/xamarin-forms/application-fundamentals/messaging-center/
+		/// 
+		/// Also see:
+		/// http://stackoverflow.com/questions/17119075/do-you-have-to-put-task-run-in-a-method-to-make-it-async
+		/// 
+		/// 
+		/// </summary>
+		/// <returns>The forever.</returns>
+		/// <param name="token">Token.</param>
+		public async Task PollForever(CancellationToken token)
+		{
+			// TODO: Disable polling when the app goes to background and we are not the host?
+
+			// TODO: Fix this warning?
+			await Task.Run(async () =>
+			{
+				long i = 0;
```

Comment:
This will be pulled out, yes. `ChangeId` will be extracted by when we "obtain a real status message" on line 39.